nodes/metric.py

```diff
@@ -509,7 +513,7 @@ def make_id(*args: str) -> str:
         goals: list[MetricDimensionGoal] = []
 
         dm = DimensionalMetric(  # Normalization or grouping is not possible at the moment.
-            id=action.id,
+            id=action.id + '_' + col.lower(),
```

Comment:
Doesn't really matter too  much but nowadays it's more common to write this like this:
`id=f'{action.id}_{col.lower()}',`